app/uploaders/image_uploader.rb

```diff
@@ -1,8 +1,6 @@
 class ImageUploader < CarrierWave::Uploader::Base
   include CarrierWave::MiniMagick
 
-  storage :aws
```

Comment:
The rake to import images was failing in development due to this line. It's not needed for any environment.